### PR TITLE
Toaster component added for success, error and warning msgs.

### PR DESCRIPTION
**Pull request by DevRaahul**
Added toaster msgs for success, error and warning conditions.
image

### CODE REVIEW

File: Toasters/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css" />
+    <link rel="stylesheet" href="./documentation.css" />
+    <link rel="stylesheet" href="./index.css" />
+    <title>Notification Toasters</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="toaster success-toaster ">
+            <span class="toaster-icon"><i class="fas fa-check-circle"></i></span>
```
Comment (prajapati-priyanka) [inline Toasters/index.html:17]: you have added the icon, but it's not showing in the screenshot you have attached. Do check that.

File: Toasters/index.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@import url("../main.css");
+@import url("../utility.css");
+
+.toaster {
+  border-radius: 0.3rem;
+  padding: 1rem 3rem 1rem 1rem;
+  font-size: 1rem;
+  margin: 1rem 0;
+  position: relative;
+  width: 100%;
```
Comment (prajapati-priyanka) [inline Toasters/index.css:10]: Just a suggestion, you can decrease the width of the toaster. I think it will look more good.

Comment (DevRaahul) [reply Toasters/index.css:10]: I have given 100% width so whenever we use a toaster in-app then we will wrap one outer div which will have its own width so the toaster will take the whole width allowed to it.

File: Toasters/index.html (hunk 1 above)
Comment (DevRaahul) [reply Toasters/index.html:17]: changed screenshot plz check.

File: Toasters/index.css (hunk 2 above)
Comment (prajapati-priyanka) [reply Toasters/index.css:10]: okay. That's good.

File: Toasters/index.html (hunk 1 above)
Comment (prajapati-priyanka) [reply Toasters/index.html:17]: yes, it is showing now.

**Review by prajapati-priyanka — APPROVED** [review]

**Review by AtulPant2704 — APPROVED** [review]

File: Toasters/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            Error toaster: Somthing went wrong please try again...!
+        </div>
+        <div class="toaster warning-toaster ">
+            <span class="toaster-icon"><i class="fas fa-exclamation-triangle"></i></span>
+            Warning toaster: Somthing went wrong please try again...!
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (VishalPatil18) [inline Toasters/index.html:31]: You could add an extra line at the end of the file to avoid such warnings by Github

**Review by VishalPatil18 — APPROVED** [review]

**Pull request merged**